static/js/stream_ui_updates.js

```diff
@@ -98,16 +98,11 @@ export function update_regular_sub_settings(sub) {
     }
     const $settings = $(`.subscription_settings[data-stream-id='${CSS.escape(sub.stream_id)}']`);
     if (sub.subscribed) {
-        if ($settings.find(".email-address").val().length === 0) {
-            // Rerender stream email address, if not.
-            $settings.find(".email-address").text(sub.email_address);
-            $settings.find(".stream-email-box").show();
-        }
         $settings.find(".personal_settings").addClass("in");
+        $settings.find(".copy_email_button").prop("disabled", false);
     } else {
         $settings.find(".personal_settings").removeClass("in");
-        // Clear email address widget
-        $settings.find(".email-address").html("");
+        $settings.find(".copy_email_button").prop("disabled", true);
```

Comment:
Do we want to hide the email address section only instead of disabling the button? @alya @timabbott thoughts?